server/chat-plugins/friends.ts

```diff
@@ -268,8 +272,8 @@ export const commands: Chat.ChatCommands = {
 				sendPM(`/uhtmlchange sent,`, targetUser.id);
 				sendPM(`/uhtmlchange undo,`, targetUser.id);
 			}
-			await Chat.Friends.cache.update(user.id);
-			await Chat.Friends.cache.update(target);
+			await Chat.Friends.updateCache(user);
```

Comment:
`updateCache` calls should probably be inside the actual handlers (i.e. inside `Friends.approveRequest`).

Reply:
See below.